.travis.yml

```diff
@@ -6,7 +6,7 @@ otp_release:
 sudo: true
 before_script:
   - sudo pip install crossbar[msgpack]
-script: mix test.all && mix spell.example.pubsub > /dev/null
+script: mix test.all && mix spell.example.pubsub && mix spell.example.rpc > /dev/null
```

Comment:
if you want to remove the output of the examples in tavis you should do
mix test.all && mix spell.example.pubsub > /dev/null && mix spell.example.rpc > /dev/null